packages/ipfs/README.md

```diff
@@ -0,0 +1,26 @@
+# @windingtree/org.id-ipfs
+ORGiD IPFS library
+
+## Documentation
+
+
+## Setup
+
```

Comment:
Here must be added a quote about node.js engine version.
```
> This package is requiring a node.js version 16 (up)
```
Also, here you have to mention that this is a private package for now and dedicated to being used in a frame of the mono repository. To add this package other packages must add the dependency:

```
"@windingtree/ipfs-apis": "1.0.0"
```